v2/emailpassword/serverless/with-aws-lambda/auth-serverless.mdx

```diff
@@ -5,16 +5,27 @@ hide_title: true
 ---
 
 <!-- COPY DOCS -->
-<!-- ./thirdpartyemailpassword/serverless/with-aws-lambda/auth-serverless.mdx -->
+<!-- ./emailpassword/serverless/with-aws-lambda/auth-serverless.mdx -->
+
+import CoreInjector from "/src/components/coreInjector"
+import BackendSDKTabs from "/src/components/tabs/BackendSDKTabs";
+import TabItem from '@theme/TabItem';
 
 # 3. Exposing Auth APIs
 
 We will add all the backend APIs for auth on `/auth/*`. This can be changed by setting the `apiBasePath` property in the `appInfo` object on the backend and frontend. For the rest of this page, we will assume you are using `/auth/*`.
 
-## Create a new file `auth.js`
+## Copy this snippet to your handler file
+<!-- TODO: Add example app for python -->
 - An example of this can be found [here](https://github.com/supertokens/supertokens-auth-react/blob/master/examples/with-aws-lambda/backend/auth.js).
 
-```tsx title="auth.ts"
+
+<CoreInjector>
+
+<BackendSDKTabs disableGolang>
+<TabItem value="nodejs">
+
```

Comment:
```suggestion

An example of this can be found [here](https://github.com/supertokens/supertokens-auth-react/blob/master/examples/with-aws-lambda/backend/auth.js).

```

Reply:
Done.